unison-syntax/src/Unison/Syntax/Lexer.hs

```diff
@@ -71,11 +71,18 @@ type BlockName = String
 type Layout = [(BlockName, Column)]
 
 data ParsingEnv = ParsingEnv
-  { layout :: !Layout, -- layout stack
-    opening :: Maybe BlockName, -- `Just b` if a block of type `b` is being opened
-    inLayout :: Bool, -- are we inside a construct that uses layout?
-    parentSection :: Int, -- 1 means we are inside a # Heading 1
-    parentListColumn :: Int -- 4 means we are inside a list starting at the fourth column
+  { -- layout stack
+    layout :: !Layout,
+    -- `Just b` if a block of type `b` is being opened
+    opening :: Maybe BlockName,
+    -- are we inside a construct that uses layout?
+    inLayout :: Bool,
+    -- Use a stack to remember the parent section and
+    -- allow docSections within docSessions.
```

Comment:
```suggestion
    -- allow docSections within docSections.
```
(was this a typo or intentional?)

Reply:
Oh well spotted!  Total typo I'm afraid :sweat_smile: